getting-started/1_template.md

```diff
@@ -0,0 +1,18 @@
+We’ll start by getting the app up and running, and then explain the different components which we will later extend.
+
+First off, open a terminal and instantiate the template project.
```

Comment:
At this point, the user has no idea what a template project is.

Reply:
This is a copy & paste job from https://docs.daml.com/getting-started/index.html. I'm all for improving it, but I'd like to keep this PR purely to the docs -> scenario conversion.